crates/rover-client/src/blocking/client.rs

```diff
@@ -0,0 +1,65 @@
+use crate::headers;
+use crate::RoverClientError;
+use graphql_client::GraphQLQuery;
+
+/// Represents a client for making http requests.
+pub struct Client {
+    api_key: String,
+    client: reqwest::blocking::Client,
+    uri: String,
+}
+
+impl Client {
+    /// Construct a new [Client] from 2 strings, an `api_key` and a `uri`.
+    /// For use in Rover, the `uri` is usually going to be to Apollo Studio
+    pub fn new(api_key: String, uri: String) -> Client {
```

Comment:
```suggestion
    pub fn new(api_key: &str, uri: &str) -> Client {
```

taking a string slice here will make our API a bit more flexible

![image](https://user-images.githubusercontent.com/9408157/96897480-41485580-1454-11eb-82fd-c08dff3998e6.png)
src https://doc.rust-lang.org/book/ch04-03-slices.html

Reply:
fixed in #50